index/circuitdojo.json

```diff
@@ -9,7 +9,14 @@
             "kind": "sample",
             "tags": ["lte"],
             "apps": "samples/*",
-            "compatibleNcs": ["v2.4.2"]
+            "releases": [
+                {
+                    "date": "2023-12-20T20:37:28Z",
+                    "name": "2.4.x",
+                    "tag": "v2.4.x",
+                    "sdk": "v2.4.1"
```

Comment:
All over it looks very inconsistent that the SDK version sometimes starts with `v` sometimes doesn't. How is it expected? Even if it works both ways, this should probably be cleaned up.

Reply:
`v` should be used for consistency with the docs throughout all of the releases, if you ask me.

Reply:
This is only data here, whether the version contains `v` or not here is not relevant if it's takes care of when we display it, but consistency and data integrity is important.